src/map/mob.c

```diff
@@ -3513,8 +3513,6 @@ static int mobskill_use(struct mob_data *md, int64 tick, int event)
 				case MSC_FRIENDSTATUSON: // friend status[num] on
 				case MSC_FRIENDSTATUSOFF: // friend status[num] off
 					flag = ((fmd = mob->getfriendstatus(md, ms[i].cond1, ms[i].cond2)) != NULL); break;
-				case MSC_SLAVELT: // slave < num
```

Comment:
same for this one, why is it being removed.

Reply:
Removed related commits.